server/.gitignore

```diff
@@ -0,0 +1,5 @@
+DS_Store
+.DS_Store
+node_modules
```

Comment:
the node_modules are still showing up. Try adding `**/node_modules`. Deleting the node_modules folder. Npm install and then committing again.